baseline.yaml

```diff
@@ -358,33 +358,34 @@ criteria:
     security_insights_value: # TODO
     scorecard_probe: # TODO
 
-  - id: OSPS-DO-01
+  - id: OSPS-GV-01
     maturity_level: 1
-    category: Documentation
+    category: Governance
     criterion: |
       The project MUST have one or more mechanisms
-      for public discussions about proposed
-      changes and usage obstacles.
+      for public discussions about project functionality.
     rationale: |
       Encourages open communication and
       collaboration within the project community,
       enabling users to provide feedback and
       discuss proposed changes or usage
       challenges.
     details: |
-      Establish one or more mechanisms for public 
+      Establish and document one or more mechanisms for public 
       discussions within the project, such as
       mailing lists, instant messaging, or issue
       trackers, to facilitate open communication
-      and feedback.
+      and feedback.  The presence of a repository-linked issue tracker,
```

Comment:
I know GitHub and GitLab are the obvious contenders in repository and SCM, but there exists situations with some of these repositories (although niche) where code is public but issue, wiki, or certain documentation not "in the tree" of check-in code is not. Do you want to clarify such example resources _must_ be configured to accept new issues and/or submissions from the public?

Reply:
This makes sense though I would maybe limit it to allowing whatever private stuff they want as long as there is a public way to do it. For example, some open source repos do have maintainer only wikis for stuff that is internal and/or sensitive like non-public security vulnerabilities.

Reply:
I'm willing to update this PR (which is just renumbering and tooling) or open a new PR for that -- the discussion on Friday pointed towards the criteria "there is at least one way for the general public to provide feedback on the software" -- I'm not sure if that included the criteria that the communication channel *also* needed to be visible to other members of the public, but that feels reasonable to me (i.e. a mailing list needs to be public / have public archives to count as such a communication channel).

My preference would be to make that change in a separate PR, to allow this one to merge more quickly.

Reply:
I think just adding the word "public" before "repository-linked" is sufficient to address this concern. If we want to do more substantive wordsmithing, then I agree with Evan that it should be a separate PR so we can land this one.